src/main/java/data/CameraTimeline.java

```diff
@@ -0,0 +1,25 @@
+package data;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Created by Bart.
+ * Class to store information about a camera timeline.
+ */
+public class CameraTimeline extends Timeline {
+
+    // The camera that is associated with this timeline.
+    @Getter @Setter
+    private Camera camera;
+
+    /**
+     * Constructor.
+     * @param camera        - the camera belonging to this timeline
```

Comment:
I think it is more common to just have the javadoc line without the spacing and `-`. I think the spacing doesn't really add much, `* @param camera the camera belonging to this timeline` would be better.

The Collections class in Java is a really good example on good and simple javadoc. [Look here](http://www.docjar.net/html/api/java/util/Collections.java.html)